src/webapp/components/collapsible-section/Section.tsx

```diff
@@ -1,10 +1,10 @@
 import { Icon, IconButton, makeStyles, Paper } from "@material-ui/core";
-import React from "react";
+import React, {useMemo} from "react";
```

Comment:
That's not prettified. `yarn prettify` does the job, but we typically prefer to configure vscode with extension Prettier + "Format on save" so we forget about it. Also, the automatic formatting is very helpful when writing code.